packages/manager-react-components/src/hooks/tags-list/useTagsFilter.tsx

```diff
@@ -0,0 +1,11 @@
+export const useTagsFilter = ({
+  tags,
+  displayInternalTags,
+}: {
+  tags: { [key: string]: string };
+  displayInternalTags: boolean;
+}) => {
+  return Object.keys(tags)
+    .filter((key) => displayInternalTags || key !== 'ovh')
+    .map((key) => `${key}:${tags[key]}`);
+};
```

Comment:
```suggestion
export const filterTags = ({
  tags,
  displayInternalTags,
}: {
  tags: { [key: string]: string };
  displayInternalTags?: boolean;
}) => Array.from(new Set(...Object.keys(tags)
    .filter((key) => displayInternalTags || key !== 'ovh')
    .map((key) => `${key}:${tags[key]}`)));
```

Array.from(new Set()) to make sure each tag appear only once

Reply:
this case isn't possible bc this tags entry is Object and with object we dont have duplicate key